rules/app.bzl

```diff
@@ -21,6 +21,7 @@ _IOS_APPLICATION_KWARGS = [
     "strings",
     "alternate_icons",
     "settings_bundle",
+    "minimum_deployment_os_version",
```

Comment:
maybe put it aside with minimum_os_version and give a comment to explain their difference?

Reply:
The docs for `ios_application` are at rules_apple https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-ios.md#ios_application  - Perhaps we can just remove this wrapper longer term.